src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -576,6 +584,9 @@ public boolean configure(final StaplerRequest req, final JSONObject formData)
       // Grab whitelist
       this.setWhitelist(formData.getString("whitelist"));
 
+      // Grab the Global Job Tags
+      this.setGlobalJobTags(formData.getString("globalJobTags"));
+
       // Grab tagNode and coerse to a boolean
       if ( formData.getString("tagNode").equals("true") ) {
```

Comment:
in case there is a null, you should do "true".equals(formData.getString("jobRegex"))